bento4/default/PKGBUILD

```
@@ -21,7 +21,7 @@ build() {
 
   sed -i "s/ STATIC / SHARED /" CMakeLists.txt
   cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX="/usr" -DCMAKE_SKIP_BUILD_RPATH=ON
-  make
+  cmake --build ${srcdir}
```

Unquoted use of "$srcdir" should be avoided. Likely it makes most sense to simply use `.` here:
```suggestion
  cmake --build .
```